src/tests/governance.rs

```diff
@@ -55,7 +58,165 @@ fn begin_dispute() {
 				Address::random()
 			));
 			assert_noop!(
-				Tellor::begin_dispute(RuntimeOrigin::signed(another_reporter), query_id, 0),
+				Tellor::begin_dispute(RuntimeOrigin::signed(another_reporter), query_id, 0, None),
+				Error::NoValueExists
+			);
+			assert_ok!(Tellor::submit_value(
+				RuntimeOrigin::signed(reporter),
+				query_id,
+				uint_value(100),
+				0,
+				query_data.clone(),
+			));
+			now()
+		});
+
+		let dispute_id = with_block(|| {
+			// await h.expectThrow(gov.connect(accounts[4]).beginDispute(ETH_QUERY_ID, blocky.timestamp)) // must have tokens to pay/begin dispute
```

Comment:
Remove comment as now implemented, retaining the `// must have tokens to pay/begin dispute` comment so it can be compared with source tests please.